t/control/healthcheck.t

```diff
@@ -48,6 +35,13 @@ run_tests;
 __DATA__
 
 === TEST 1: upstreams
+--- yaml_config
+apisix:
+    node_listen: 1984
+deployment:
+    role: data_plane
+    role_data_plane:
+        config_provider: yaml
```

Comment:
we can't keep this in the `add_block_preprocessor`?

Reply:
This test file does not use yaml_config at all.